### ISSUE TITLE
Reference collection

### ISSUE THREAD

**Issue by JaneGodiksen** [ices-eg/SmartDots#97]
SmartDots needs development to facilitate agreed aged reference collections.

An area for reference collections, where everyone has access to test themselves – not invited by a coordinator. One will only be able to see the “correct” ages and one own annotations. Not the other users who have tested themselves on the dataset.

**Comment by JCoadDavies** [comment]
Some options discussed 
- A new type of event can be created called "Reference Collection" with a collection of agreed age fish from a workshop. These type of events can be filterable. The coordinator/expert would upload and annotate the images.
- The AQ codes could have a new code "Agreed Age" and these are filterable and possibly extracted to a new event including all agreed age images for that species
- This could be done in the web application on view event page

We agreed to think further on this and come back to at our next meeting

**Comment by JCoadDavies** [comment]
This was taken up again at the October 23rd WebEx 2020 and it was agreed that it would be very useful to have a new type of "Reference" event. Such an event would only include samples with a validated stage/age or agreed age (stage?) or eggs and larvae with true ID. These would be publicly available.

**Comment by JaneGodiksen** [comment]
Adrian and Mandy to be assigned. I can't find them in the list. Maybe they don't have access?

**Comment by cmspinto** [comment]
These specifications are like the public events?

[
![Uploading image.png…]()
](https://smartdots.ices.dk/ViewEvent?key=74)


Where anyone can see this events:

![image](https://user-images.githubusercontent.com/8705477/106598402-f6ac9e00-6557-11eb-9434-a762a7c8637e.png)


And anyone can make annotations and they can only see their annotations:

![image](https://user-images.githubusercontent.com/8705477/106598364-e7c5eb80-6557-11eb-9fcc-3766acc4eede.png)
 

So this feature might already exists and just needs some tweaks!

**Comment by JCoadDavies** [comment]
Yes, anyone can see those events so you are right Carlos we can use this and just tweak it a bit

**Comment by mandygault** [comment]
Aspects to be considered and agreed in the creation of the reference collection 

A reference collection can be defined as a collection of prepared ageing structures, of known consensus-derived ages, representative of all factors that might reasonably be expected to influence the appearance or relative size of the growth increments (Campana, 2001)

The purpose of a reference collection is to have a set of otoliths as reference material for calibration and training of new and established readers and to reach consensus on the interpretation of the otolith characteristics (ICES Co-operative Research Report No 346 “Handbook of Fish Ageing Estimation Protocols and Validation Methods”)

What factors might reasonably influence the appearance or relative size of growth increments?
Temporal
Spatial
Numbers suggested in the region of 200 otoliths, although Campana notes that ability to memorise ages means that up to 500 may be beneficial. 

The study group on Baltic cod age reading (ICES CM 2000/H:01) outlines different types of reference collections: Control collection , Agreed collection
This report also includes information on the number of otoliths used in their reference collection, and how the collection may be updated.

Number per age stratification?
Stratifications would need to be agreed,  dependent on max age range per species.  
	
NOAA Fisheries include information on reference sets including: Consensus between (at least) two in-house readers and an external reader, as part of regular exchanges. 
*Suggest an agreement between a minimum of 3 readers, no more than 2 from one institute. 
To be agreed

We need to look at  standardised processes covering ageing and possibly photography (may need to deal with more than one method, sectioned / whole / burning).  Do we need to cover different methods?  
What is appropriate magnification – e.g. reading older otoliths sometimes requires higher magnification.   

Good agreed images can be extracted from existing exchanges on SmartDots to facilitate the reference collection for some species. Other species not already in a SmartDots event will need to have exchanges organised specifically for the reference collection? 

We would need to ensure we cover each area under the temporal requirements. 
Exchange co-ordinators to co-ordinate?
Do we need to keep hard otoliths too?

Institutes that hold known age otoliths, e.g. mark recapture could be approached to upload.

**Comment by JaneGodiksen** [comment]
Comment for Q1 meeting: 
For agreement it is suggested three readers, and only two from one institute. How about agreement is done during a workshop? Only otoliths that are agreed by the participants can be included?
Method: We need to cover the methods used in the exchanges.

**Comment by JCoadDavies** [comment]
[SMARTDOTS reference collection guidance draft.docx](https://github.com/ices-eg/SmartDots/files/8521168/SMARTDOTS.reference.collection.guidance.draft.docx)

**Comment by JCoadDavies** [comment]
Mandy, Adrian and Gudrun unable to attend Q2 2022 WGSMART meeting so a meeting needs to be organised to plan next steps

**Comment by wimallegaert** [comment]
Question: in order to arrive at a reference collection, both samples and annotations that qualify for the reference collection must be able to be marked. In which software is this functionality best added? The web application of the SmartDots software?

**Comment by UlrikaBeier** [comment]
> Question: in order to arrive at a reference collection, both samples and annotations that qualify for the reference collection must be able to be marked. In which software is this functionality best added? The web application of the SmartDots software?
As I understand it, you (Wim) mean samples in existing events on SmartDots@ICES, could be marked e.g. after a completed workshop as being qualified for a reference collection?
(Whether it is suitable in the web application or in the SmartDots software, I cannot judge.)

FYI - A small group formed at WKARP2 have produced an example of a reference set from WKARP2 (event 402), where a subset of the samples with e.g. 100% agreement have now been added to a separate event (409). Based on this exercise, the group will comment on the current Draft Guidance for SMARTDOTS Reference Collections, including a list of proposed/desired functionalities for setting up and using test sets, this to be brought forward to WGSMART and WGBIOP.

**Comment by kbekaert** [comment]
Wish list is needed from WGBIOP.

**Comment by JCoadDavies** [comment]
Notes from WKARP2 which were added to by WGSMART

[Plaice reference collection event to BIOP_WGSMART.docx](https://github.com/ices-eg/SmartDots/files/9852551/Plaice.reference.collection.event.to.BIOP_WGSMART.docx)

**Comment by wimallegaert** [comment]
Related to issue https://github.com/ices-eg/SmartDots/issues/301

**Comment by cmspinto** [comment]
This is already done:

![Image](https://github.com/user-attachments/assets/063ff79c-5254-41f0-8bf3-71761ff0f863)
